freqtrade/freqai/prediction_models/ReinforcementLearner.py

```diff
@@ -71,7 +71,8 @@ def fit(self, data_dictionary: Dict[str, Any], dk: FreqaiDataKitchen, **kwargs):
 
         model.learn(
             total_timesteps=int(total_timesteps),
-            callback=[self.eval_callback, self.tensorboard_callback]
+            callback=[self.eval_callback, self.tensorboard_callback],
+            progress_bar=self.rl_config.get('progress_bar', False)
```

Comment:
We do use progressbars in freqtrade in other places - and i know that it tends to mess up regular output (and requires quite some messing around).

which output (stdout / stderr, ...) is this outputting?

How will it impact actual logfiles (`--log-file somelog.log`)?

Reply:
The progress bar has no impact on the log files using `--log-file`.
All ft messages are displayed correctly, it seems to me.
But I have only checked linux native console output. 

However, when new messages arrive, the progress bar leaves its old copy on the screen.
It's not a big problem for me, but I can understand if this is unacceptable to freqtrade.

See screenshot:
<img width="584" alt="scr" src="https://user-images.githubusercontent.com/37817561/229367366-a0de1f9b-4913-4450-9c34-0396db759dfc.png">

Reply:
this is unfortunately mostly what i expected, and makes it pretty much useless if you ask me.

Reply:
inspired by this (rich progressbars just look so much nicer) , i've created #8461 - using rich as progressbar for hyperopt.

Some of the changes that required also impacted logging compatibility - which i suspect MAY benefit this PR as well.

@initrv, Would you be so kind to re-test the progressbar behavior with the updated PR (I've pushed a merge from develop for this) - basically doing the same test than in the screenshot above.
I'd hope that the progressbar now stays "at the bottom" - no longer re-rendering due to log messages (unless you press random keys ... ) - but didn't test that so far.

Reply:
That's great, thanks @xmatthias ! I confirm that the progress bar now works much better and stays at the bottom until training finished. As long as you don't resize the terminal window or press keys :)

Reply:
The resizing is a ... different problem - but honestly, not one i care about :laughing:

Same with pressing keys in a "read only" terminal - shoot yourself in the foot? sure, go on - we won't stop you :laughing: